website/docs/monitors/0400-DCGMI.md

```diff
@@ -0,0 +1,92 @@
+# DCGMI
```

Comment:
Thank you for the document, please add corresponding row to overview/overview.md with license link